### ISSUE TITLE
Unable to add waypoints in mission for ground rover vehicles

### ISSUE THREAD

**Issue by Jaeyoung-Lim** [mavlink/qgroundcontrol#10385]
## Expected Behavior
Be able to plan a  mission by adding waypoints.

## Current Behavior
When trying to add a waypoint to a ground rover in the plan view, it is not possible to add a waypoint since the waypoint button is greyed out.

## Steps to Reproduce:
1. Connect QGC to a ground rover vehicle
2. Enter plan view
3. Not possible to add waypoints since the waypoint button is greyed out

## System Information
When posting bug reports, include the following information
- Operating System: Ubuntu 20.04
- QGC Version: 4.2.1
- QGC build: stable
- Flight Controller: SITL Gazebo
- Autopilot (with version): PX4 `v1.13.0` and `main`

## Detailed Description
It seems that for ground rover vehicles it is not possible to add waypoints and therefore not possible to plan a mission (at all)

## Log Files and Screenshots
![image](https://user-images.githubusercontent.com/5248102/187669683-3fb11b85-29f2-4531-860d-145fc201a6b5.png)

Note that the buttons are greyed out, and therefore no waypoints can be added

**Comment by mvp-cwlin** [comment]
In the Application Setting, try to enable "Mission Do Not Require Takeoff Item"

![Capture](https://user-images.githubusercontent.com/71658930/187767849-61bdc603-c4f6-4956-9c8f-156105208112.PNG)

**Comment by Jaeyoung-Lim** [comment]
@mvp-cwlin Thanks, but shouldn't this not be necessary, since rovers are able to handle takeoff waypoints?

**Comment by mvp-cwlin** [comment]
If you change the vehicle type setting on QGroundControl to Multi-Rotor, it will let you add a takeoff item, then you can add waypoints. And yes, I believe rovers can handle takeoff.

**Comment by Jaeyoung-Lim** [comment]
@mvp-cwlin I am not sure if I agree

QGC already knows which vehicle type it is so it should not prevent the user from planning a mission

**Comment by mvp-cwlin** [comment]
I agree, QGC should automatically configure to not require a takeoff item when it knows it's a rover.
